automation_tools/scripts/ga-migration/main.py

```diff
@@ -238,11 +241,24 @@ def migrate_repo(path):
     # Remove bak files
     delete_file(path + "*.bak")
 
-    # setup.py: replace pytest deps with pytest-invenio
+    # Simplify setup.py test requirements replacing them with pytest-invenio
     replace_list(
         path + "setup.py",
         r"tests_require = (['\"\'[\s*\"(a-z-A-Z><=0-9.\[\]),]*])",
-        ["pytest-cov", "pytest-pep8"],
```

Comment:
Minor: `pytest-pep8` should remain as we want to get rid of it.
We also need to add others as `isort` I believe..